concordia/static/js/base.js

```diff
@@ -85,11 +85,15 @@ $(function() {
             'You are using an outdated browser. This website fully supports the current ' +
             'version of every major browser ' +
             '(Microsoft Edge, Google Chrome, Mozilla Firefox, and Apple Safari). See ' +
-            'our <a href="/help-center/#headingTwelve">browser support policy</a> ' +
+            'our <a href="/help-center/#browserSupport">browser support policy</a> ' +
             'for more information.';
 
         displayHtmlMessage('danger', theMessage);
     }
+
+    if (location.hash && $('#faqAccordion')) {
```

Comment:
I think `$('#faqAccordion')` will always evaluate to true. This could be a simple length check:

```suggestion
    if (location.hash && $('#faqAccordion').length > 0) {
```

Alternatively, you could just use jQuery to scope it implicitly since the only elements we care about should all be children of the accordion element anyway: 

```javascript
if (location.hash) {
    $('#faqAccordion').find(location.hash).collapse('show');
}
```

Reply:
The second option didn't work so I used the first option. Tested in the debugger.